cmd/operator/main.go

```diff
@@ -282,8 +275,34 @@ func Main() int {
 		return 1
 	}
 
+	var scrapeConfigSupported bool
+	err = cc.CheckPrerequisites(ctx,
```

Comment:
L279-300 could be refactored into a small function to reduce duplication. E.g.

````golang
...
ok, err := checkPrequisites(ctx, logger, cc, verbs, ..., ...)
if err != nil {
    ...
    return 1
}

...

func checkPrequisites(ctx context.Context, logger log.Logger, cc *k8sutil.CRDChecker, verbs map[string][]string, sgv, resource string) (bool, err) {
...
```

Reply:
Done